benchmark/src/main/java/io/servicefabric/transport/protocol/ProtostuffMessageSerializerBenchmark.java

```diff
@@ -0,0 +1,109 @@
+package io.servicefabric.transport.protocol;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.ThreadLocalRandom;
+import java.util.concurrent.TimeUnit;
+
+import org.openjdk.jmh.annotations.*;
+
+import com.google.common.collect.ImmutableMap;
+
+import io.netty.buffer.ByteBuf;
+import io.netty.buffer.Unpooled;
+import io.servicefabric.cluster.gossip.Gossip;
+import io.servicefabric.cluster.gossip.GossipRequest;
+import io.servicefabric.transport.TransportHeaders;
+
+@Fork(2)
+@State(Scope.Thread)
+@Threads(4)
+@Warmup(iterations = 5)
+@Measurement(iterations = 5)
+@BenchmarkMode(Mode.AverageTime)
+@OutputTimeUnit(TimeUnit.NANOSECONDS)
+public class ProtostuffMessageSerializerBenchmark {
```

Comment:
Benchmark serialization/deserialization results; ser/deser is scalable despite message size, except case ```gossipReq```, strange it takes significant time comparing to others. In the test ```gossipReq``` was ```GossipRequest``` populate with x5 regular(256b) ```Gossip``` messages, so in total it was < 1k.